test/unit/controllers/declaration/RoutingCountriesSummaryControllerSpec.scala

```diff
@@ -281,4 +281,18 @@ class RoutingCountriesSummaryControllerSpec extends ControllerSpec {
       }
     }
   }
+
+  "Routing countries controller" should {
+
+    "handle duplication during changing the country" in {
+
+      withNewCaching(aDeclaration(withRoutingCountries(Seq("PL", "DZ", "AD"))))
+
+      val duplicatedCountry = JsObject(Seq("country" -> JsString("PL")))
+
+      val result = controller.submitChangeCountry(Mode.Normal, "DZ")(postRequest(duplicatedCountry))
```

Comment:
So in this request, DZ is the country I want to change, and I want to change it to PL (which is already in the list - hence error).  Is that right?

Reply:
Yes, exactly. I can add some local variables to describe those countries if you want